gdal/ogr/ogrsf_frmts/mdb/ogrmdbjackcess.cpp

```diff
@@ -162,9 +167,41 @@ int OGRMDBJavaEnv::Init()
     {
         JavaVM* vmBuf[1];
         jsize nVMs;
+        int ret = 0;
+
+#if JVM_LIB_DLOPEN
+        const char *jvmLibPtr = "libjvm.so";
+        char jvmLib[PATH_MAX];
+
+        FILE *javaCmd = popen("\"${JAVA_HOME}${JAVA_HOME:+/bin/}java\" -XshowSettings 2>&1 | sed -n '/\\bsun\\.boot\\.library\\.path =/s:.* = \\(.*\\):\\1/server/libjvm.so:p'", "r");
```

Comment:
Hum, this is rather impressive black magic :-) I'm wondering if we couldn't make that less Unix specific (by the way is it well portable over various shells, or on MacOS...), and perhaps even Windows portable by using the API in port/cpl_spawn.h just to launch java -XshowSettings, get the content of stderr and then parse back the result in the GDAL code ? I'm aware this would be a bit of work and you probably don't care about Windows uses, so I won't argue too strongly about that. But at least some comment to document what this line does would be welcome (basically your commit message)